src/api/Folder.js

```diff
@@ -11,7 +11,15 @@ import { getBadItemError } from '../utils/error';
 import Item from './Item';
 import FileAPI from './File';
 import WebLinkAPI from './WebLink';
-import { CACHE_PREFIX_FOLDER, ERROR_CODE_FETCH_FOLDER, ERROR_CODE_CREATE_FOLDER } from '../constants';
+import {
+    CACHE_PREFIX_FOLDER,
+    ERROR_CODE_FETCH_FOLDER,
+    ERROR_CODE_CREATE_FOLDER,
+    FIELD_REPRESENTATIONS,
+} from '../constants';
+
+const DEFAULT_THUMBNAIL_DIMENSIONS = '1024x1024';
+const SHOULD_FETCH_THUMBNAILS = false;
```

Comment:
I'm still investigating how I can use `feature-checking` to determine this value.  This should not be a constant!